[2025-03-01T15:06:34.226+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-01T15:06:34.388+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2021-08-21T00:00:00+00:00 [queued]>
[2025-03-01T15:06:34.619+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2021-08-21T00:00:00+00:00 [queued]>
[2025-03-01T15:06:34.629+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-01T15:06:34.893+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2021-08-21 00:00:00+00:00
[2025-03-01T15:06:34.966+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'first_task', 'scheduled__2021-08-21T00:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpwaededkm']
[2025-03-01T15:06:34.963+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=453) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-01T15:06:34.981+0000] {standard_task_runner.py:72} INFO - Started process 470 to run task
[2025-03-01T15:06:34.967+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask first_task
[2025-03-01T15:06:35.507+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.first_task scheduled__2021-08-21T00:00:00+00:00 [running]> on host 93ff5a63b4d9
[2025-03-01T15:06:36.486+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='matthew' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-21T00:00:00+00:00'
[2025-03-01T15:06:36.490+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-01T15:06:36.769+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-01T15:06:36.783+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2025-03-01T15:06:36.902+0000] {subprocess.py:99} INFO - Output:
[2025-03-01T15:06:36.908+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2025-03-01T15:06:36.912+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-01T15:06:37.000+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-01T15:06:37.010+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=first_task, run_id=scheduled__2021-08-21T00:00:00+00:00, execution_date=20210821T000000, start_date=20250301T150634, end_date=20250301T150637
[2025-03-01T15:06:37.256+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-01T15:06:37.355+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T15:06:37.406+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
